src/2980R0_a_motivation_scope_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -11,7 +11,7 @@ author:
     email: <dominik.berner@gmail.com>
   - name: Johel Ernesto Guerrero Peña
     email: <johelegp@gmail.com>
-  - name: Charles Hogg
+  - name: Chip Hogg
```

Comment:
It's still Charles in the references of D2980R0 and D2981R0, as well as the "biography".

Reply:
Fixed the references to consistently use "Chip".

Reply:
Thank you!

TBH, I go back and forth as to which one I want to use in publications.  :upside_down_face: